src/runner/index.js

```diff
@@ -377,11 +377,17 @@ export default class Runner extends EventEmitter {
         return this;
     }
 
-    screenshots (path, takeOnFails, pattern) {
+    screenshots (...options) {
+        let [path, takeOnFails, pathPattern, fullPage] = options;
```

Comment:
I think we shouldn't support specifying `fullPage` as a 4th argument, only as a part of an object parameter. In this way, we will force users to refactor their code if they want to use the new feature. I think it's a good move since we will deprecate the old API one day (I hope).